### DISCUSSION TITLE
POSTAL NOT SENDING EVER SINCE CREATED

### DISCUSSION THREAD

**Discussion by DrainsD** [postalserver/postal#1707] [Help with using Postal]
Hi, I used **digital ocean** to deploy my Vps and **name cheep** as my domain, I'm new to postal, after successful installation of postal and configuring it postal not sending at all keeps saying "**SOFT FAIL,**" 
![Screenshot_1](https://user-images.githubusercontent.com/94163205/141609054-08e0e188-d901-4a34-82aa-8f14cd500837.png)
![Screenshot_2](https://user-images.githubusercontent.com/94163205/141609061-f6739529-4919-48cf-b112-cbf8a62c38f1.png)
![Screenshot_3](https://user-images.githubusercontent.com/94163205/141609062-91f2a5b2-b625-481d-89db-6d034faf7ade.png)

**Comment by JasonDevTech** [comment]
The domain you sent the email to does not have MX records set up, therefore they cannot handle mail.

**Reply by DrainsD** [reply]
but i sent to gmail too but it failed

**Comment by willpower232** [comment]
Its likely that Digital Ocean is blocking your connections to other SMTP servers which I think is the case for new accounts nowadays.

**Reply by JasonDevTech** [reply]
Even that, but in this case, bold99.com is the domain and mail-tester.com is the receiving domain. The only time I had this happen was where the receiving domain had no MX records.

**Reply by DrainsD** [reply]
how can i fix this? if digital ocean is the problem

**Reply by willpower232** [reply]
You can contact them to confirm if your emails are being blocked by them and if they are unwilling to unblock your emails then you would have to find an alternative provider

**Reply by DrainsD** [reply]
will google cloud work?

**Reply by JasonDevTech** [reply]
No, most providers block port 25 by default. Digital ocean requires you to be with them for at least two months and in good standing before you can contact them about unblocking port 25.
